functional-test/src/test/groovy/org/rundeck/util/gui/pages/jobs/JobShowPage.groovy

```diff
@@ -136,10 +136,12 @@ class JobShowPage extends BasePage implements ActivityListTrait {
         }
     }
 
+
```

Comment:
Please, avoid introducing unnecessary formatting changes. It adds noise to a PR w/o providing value.

Reply:
Revert these unneeded changes